app/javascript/react/test/DropDownSpec.js

```diff
@@ -0,0 +1,22 @@
+import DropDown from '../src/DropDown'
+import testHelper from './testHelper'
+
+
+describe('DropDown', () => {
+  let errors,
+      wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(<DropDown
+      name="state"
+      content="MA"
+      nameText="State:"
+                    />
+    );
+  });
+
+
+  it('should render a label tag', () =>{
+    expect(wrapper.find('label')).toBePresent();
+  });
+});
```

Comment:
Very happy to see this and other presentational component tests in this PR. They're a little under-built, but they're a nice starting point.